USFMConverter/Core/Render/RenderDocx.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using USFMConverter.Core.ConstantValue;
+using USFMConverter.Core.Data;
+using USFMToolsSharp.Models.Markers;
+using USFMToolsSharp.Renderers.Docx;
+
+namespace USFMConverter.Core.Render
+{
+    public class RenderDocx : RenderDocument
```

Comment:
This class is confusingly named.  It should be named something like "DocxDocument".